README.md

```diff
@@ -1,223 +1,350 @@
-# STRATUM
+# Stratum V2 (Sv2) Proxy
```

Comment:
Is misleading calling this project Proxy cause it contains a Proxy and also other things